Documents/PowerShell/profile.ps1

```
@@ -0,0 +1,5 @@
+# Starship prompt
+Invoke-Expression (&starship init powershell)
```

PSScriptAnalyzer (1.25.0)

[warning] Invoke-Expression is used. Please remove Invoke-Expression from script and find other options instead.

(PSAvoidUsingInvokeExpression)